tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# mbed-build.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2017, ARM Limited, All Rights Reserved
+#
+# Purpose
+#
+# To run test builds of the mbed OS module for all supported targets.
+
+set -eu
+
+MBED_APP='mbed-os-example-tls'
+CPUS=$(grep -c ^processor /proc/cpuinfo)
```

Comment:
I do not think this is portable across systems. I am not sure there is a portable way to do this.